modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -13,11 +13,15 @@ If you want to use only the default internal {product-title} Elasticsearch insta
 ==== 
 
 An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
-
 An output can be either:
 
-* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster, specified by server name or FQDN, and/or the internal {product-title} Elasticsearch instance. 
-* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd *forward* plug-ins.
+* `elasticsearch` to forward logs to an external Elasticsearch v5.x cluster and/or the internal {product-title} Elasticsearch instance. 
+* `forward` to forward logs to an external log aggregation solution. This option uses the Fluentd *forward* plug-ins. 
+
+[NOTE]
+====
+The endpoint must be a server name or FQDN, not an IP Address.
```

Comment:
@mburke5678  @jcantrill   "Must be"?  The IP Address works in my testing.

Reply:
According to https://bugzilla.redhat.com/show_bug.cgi?id=1825923#c10.  
It should be 'The endpoint must be a server name or FQDN, not an IP Address **when cluster-wide proxy are enabled**.'

Reply:
You seem to be right. I was looking at comment 12:
_It's indicating that it should be a server name of FQDN, but I believe that it should be more clear in a Warning indicating that it shouldn't be used an IP Address._

I made the change as you suggested.